### ISSUE TITLE
loss during training

### ISSUE THREAD

**Issue by SiyuanHuangSJTU** [orionw/rank1#3]
Hi, thanks a lot for your great work! I think exploring test-time scaling for rerankers is a very meaningful direction.

I have a quick question about the training procedure described in your paper. I did not see much detail on the loss function used during model training (apologies if I missed it). May I ask if you are using the standard teacher forcing cross-entropy loss for training, similar to traditional supervised fine-tuning (SFT)?

Additionally, have you experimented with using a “soft score” like the one in DeepSeek-R1 (i.e., true_score / (true_score + false_score)) for knowledge distillation? For example, using mean squared error loss with this kind of soft label?

**Comment by orionw** [comment]
Hi @SiyuanHuangSJTU! Yes we did not include much detail because it is just the normal language modeling loss (ie predict the next token loss).  

We used LlamaFactory and the configs are in the "train_config" folder.

Agree that there is likely many ways to improve it and looking at the loss function seems a natural choice!
